src/iop/filmicrgb.c

```diff
@@ -3591,7 +3591,7 @@ static gboolean area_scroll_callback(GtkWidget *widget, GdkEventScroll *event, g
     }
   }
 
-  return FALSE;
+  return TRUE; // Ensure that scrolling cannot move side panel when no delta
```

Comment:
You can probably remove the other `return TRUE` if you're doing going to always handle the scroll event but I'm not sure this is a good idea. I think the better solution is to only return TRUE when the Ctrl modifier is used (put the `dt_modifier_is` if statement outside of the `dt_modifier_get_scroll_unit_deltas` if statement).

Reply:
Changed as suggested. Thank you for the help.